_tools/release/02_create_pr.ts

```diff
@@ -30,14 +30,17 @@ const openedPr = await octoKit.request("POST /repos/{owner}/{repo}/pulls", {
 });
 console.log(`Opened PR at ${openedPr.data.url}`);
 
+const TARGET = "../deno/target/release/deno";
+
 function getPrBody() {
   let text = `Bumped version for ${newVersion}\n\n` +
     `Please ensure:\n` +
     `- [ ] Version in version.ts is updated correctly\n` +
     `- [ ] Releases.md is updated correctly\n` +
-    `- [ ] All the tests in this branch have been run against the CLI release being done ` +
-    `(\`../deno/target/release/deno task test && ../deno/target/release/deno task node:unit && ` +
-    `../deno/target/release/deno task node:test\`)\n\n` +
+    `- [ ] All the tests in this branch have been run against the CLI release being done` +
+    "```shell" +
+    `${TARGET} task test && ${TARGET} task node:unit && ${TARGET} task node:test` +
+    "```\n" +
```

Comment:
```suggestion
    "      ```shell" +
    `      ${TARGET} task test && ${TARGET} task node:unit && ${TARGET} task node:test` +
    "      ```\n" +
```